### ISSUE TITLE
after deleting ingress, ingress service re-appears in rancher ui

### ISSUE THREAD

**Issue by aemneina** [rancher/rancher#15288]
**Rancher versions:**
rancher/server or rancher/rancher: 1.6.18

**Infrastructure Stack versions:**
healthcheck:
ipsec:
network-services:
scheduler:
kubernetes (if applicable):

**Docker version: (`docker version`,`docker info` preferred)**

**Operating system and kernel: (`cat /etc/os-release`, `uname -r` preferred)**

**Type/provider of hosts: (VirtualBox/Bare-metal/AWS/GCE/DO)**

**Setup details: (single node rancher vs. HA rancher, internal DB vs. external DB)**
HA

**Steps to Reproduce:**
Here is Ingress control log which is showing ingress is deleted and also showing lb created within minutes. Rancher is creating Ingress after deleted via kubectl cli
```
8/28/2018 2:21:42 PMtime="2018-08-28T18:21:42Z" level=info msg="Event(api.ObjectReference{Kind:\"Ingress\", Namespace:\"my-ns\", Name:\"my-ingress\", UID:\"fdf59274-a7de-11e8-9229-020a529960bb\", APIVersion:\"extensions\", ResourceVersion:\"69027707\", FieldPath:\"\"}): type: 'Normal' reason: 'DELETE' my-ns/my-ingress"
8/28/2018 2:21:42 PMtime="2018-08-28T18:21:42Z" level=info msg="Deleting lb service [my-ns-lb-my-ingress]"
```
----
```
8/28/2018 2:22:59 PMtime="2018-08-28T18:22:59Z" level=info msg="Creating lb service [%!s(*client.LoadBalancerService=<nil>)]"
8/28/2018 2:23:06 PMtime="2018-08-28T18:23:06Z" level=info msg="Updating Rancher LB with the new lbConfig [&{{ map[] map[]} [] defaults\nlog 127.0.0.1:8514 local0 info\noption httplog\ntimeout client 50000\ntimeout connect 5000\ntimeout server 50000 1c33 [{{ map[] map[]} %!s(int64=0) https 1s8167 %!s(int64=10081) %!s(int64=8443)}] %!s(*client.LoadBalancerCookieStickinessPolicy=<nil>)}] "```

**Comment by soumyalj** [comment]
Tested with v1.6.22-rc3 on a HA setup. Created K8s environment on the HA setup. 
Created pods and ingress . Deleted the ingress. Ingress was not re-created . The steps were repeated several times and the issue did not reproduce.
Ingress controller logs:
```
9/5/2018 4:52:48 PMtime="2018-09-05T23:52:48Z" level=info msg="Updating ingress default/ingress3 with IP 165.227.76.120"
9/5/2018 4:52:48 PMtime="2018-09-05T23:52:48Z" level=info msg="Event(api.ObjectReference{Kind:\"Ingress\", Namespace:\"default\", Name:\"ingress3\", UID:\"c81dee96-b166-11e8-b1f1-02ae1e206d60\", APIVersion:\"extensions\", ResourceVersion:\"8004\", FieldPath:\"\"}): type: 'Normal' reason: 'UPDATE' default/ingress3"
9/5/2018 4:52:48 PMtime="2018-09-05T23:52:48Z" level=info msg="Event(api.ObjectReference{Kind:\"Ingress\", Namespace:\"default\", Name:\"ingress3\", UID:\"c81dee96-b166-11e8-b1f1-02ae1e206d60\", APIVersion:\"extensions\", ResourceVersion:\"7997\", FieldPath:\"\"}): type: 'Normal' reason: 'CREATE' ip: 165.227.76.120"
9/5/2018 4:52:48 PMtime="2018-09-05T23:52:48Z" level=info msg="Updating Rancher LB with the new lbConfig [&{{  map[] map[]} []   [{{  map[] map[]}    %!s(int64=0) http  1s31 %!s(int64=85) %!s(int64=80)}] %!s(*client.LoadBalancerCookieStickinessPolicy=<nil>)}] "
9/5/2018 4:53:31 PMtime="2018-09-05T23:53:31Z" level=info msg="Event(api.ObjectReference{Kind:\"Ingress\", Namespace:\"default\", Name:\"ingress3\", UID:\"c81dee96-b166-11e8-b1f1-02ae1e206d60\", APIVersion:\"extensions\", ResourceVersion:\"8068\", FieldPath:\"\"}): type: 'Normal' reason: 'UPDATE' default/ingress3"
9/5/2018 4:53:31 PMtime="2018-09-05T23:53:31Z" level=info msg="Event(api.ObjectReference{Kind:\"Ingress\", Namespace:\"default\", Name:\"ingress3\", UID:\"c81dee96-b166-11e8-b1f1-02ae1e206d60\", APIVersion:\"extensions\", ResourceVersion:\"8070\", FieldPath:\"\"}): type: 'Normal' reason: 'DELETE' default/ingress3"
9/5/2018 4:53:31 PMtime="2018-09-05T23:53:31Z" level=info msg="Deleting lb service [default-rancherlb-ingress3]"


```

**Comment by alena1108** [comment]
Fixed in rancher/lb-service-rancher:v0.9.7

**Comment by sangeethah** [comment]
Setup:

Fresh install of v1.6.18-patch1-rc2 .
Created K8s env . k8s version - k8s 1.10.11-rancher1-2.
Manually updated ingress controller in Kunbernetes service to rancher/lb-service-rancher:v0.9.8-rc2

Delete 10 existing ingress
Create 10 ingress.

Repeated the above 2 steps multiple times in parallel.

rancherlb services relating to the ingresses get removed for all the deleted ingresses.
